pythran/pythonic/numpy/interp_core.cpp

```diff
@@ -0,0 +1,256 @@
+//
+//  interp_core.cpp
+//  From /Users/jlaroche/temp/NumpySrc/numpy/core/src/multiarray/compiled_base.c
```

Comment:
Well, the licencing should be reproduced here.

Reply:
There isn't any licensing in the original file.